### PR TITLE
Two Fixes

### PR DESCRIPTION
**Pull request by Morritz**
1. WebContainers support for polyfill
2. Fixing Typescript error in one iteration loop.

### CODE REVIEW

File: src/polyfill.ts [hunk 1]
```diff
@@ -215,21 +215,21 @@ export const polyfill = (target: any, options?: PolyfillOptions) => {
 	) as Set<string>
 
 	// expand exclude options using exclusion shorthands
-	for (const excludeOption of excludeOptions) {
+	excludeOptions.forEach(function(excludeOption){
```
Comment (jonathantneal) [inline src/polyfill.ts:218]: The current code avoids creating functional scope, which is preferred.

File: src/polyfill.ts [hunk 2]
```diff
@@ -215,21 +215,21 @@ export const polyfill = (target: any, options?: PolyfillOptions) => {
...
 		if (excludeOption in exclusions) {
 			for (const exclusion of exclusions[excludeOption as keyof typeof exclusions]) {
 				excludeOptions.add(exclusion)
 			}
 		}
-	}
+	});
```
Comment (jonathantneal) [inline src/polyfill.ts:224]: I think prettier would have caught this, but this project has an over-opinionated position of avoiding semicolons. 😉😅

File: src/polyfill.ts (hunk 1 above)
Comment (Morritz) [reply src/polyfill.ts:218]: I think then it could be just the same but with .values() at the end or explicitly as "something"

Comment (jonathantneal) [reply src/polyfill.ts:218]: So, in TS, `excludeOptions` is typed as `Set<string>`. As a kind of [iterable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/@@iterator), `Set` is ideal for this kind of loop.

This kind of iteration is explicitly demonstrated on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/@@iterator#using_iterator_with_for..of).

```js
const mySet = new Set();
mySet.add('0');
mySet.add(1);
mySet.add({});

for (const v of mySet) {
  console.log(v);
}
```

Comment (Morritz) [reply src/polyfill.ts:218]: image
Do you have this flag set to enabled or just perhaps newer typescript version knows that it is good to write that like this?

Comment (jonathantneal) [reply src/polyfill.ts:218]: You might have an older TS.

**Comment by jonathantneal** [comment]
Closing this, as none of the changes can be accepted.

**Pull request closed without merging**